chain/network/src/network_protocol/borsh.rs

```diff
@@ -1,12 +1,11 @@
-/// Contains types that belong to the `network protocol.
-///
-/// WARNING WARNING WARNING
-/// WARNING WARNING WARNING
-/// We need to maintain backwards compatibility, all changes to this file needs to be reviews.
+//! Contains types that belong to the `network protocol.
+//!
+//! WARNING WARNING WARNING
+//! WARNING WARNING WARNING
+//! We need to maintain backwards compatibility, all changes to this file needs to be reviews.
+use crate::network_protocol::edge::{Edge, PartialEdgeInfo};
+use crate::network_protocol::{PeerChainInfoV2, PeerInfo, RoutedMessage};
```

Comment:
should we kill borsh already?

Reply:
I tried recently, but our python test suite is very messy and it would take me a few weeks to remove borsh usage there. I don't have that capacity now.